downstream/assemblies/hub/assembly-populate-container-registry.adoc

```diff
@@ -12,6 +12,24 @@ ifdef::context[:parent-context: {context}]
 [role="_abstract"]
 By default, private {HubName} does not include container images. To populate your container registry, you need to push a container image to it. The procedures in this section describe how to pull images from the Red Hat Ecosystem Catalog (registry.redhat.io), tag them, and push them to your private {HubName} container registry.
 
+[IMPORTANT]
+====
+Image manifests and filesystem blobs are served directly from `registry.redhat.io` and `registry.access.redhat.com`. 
+However, from 1st of May 2023, filesystem blobs are served from `quay.io` instead. 
+To avoid problems pulling container images, you must enable outbound connections to the following hostnames:
+
+* `cdn.quay.io`
+* `cdn01.quay.io`
+* `cdn02.quay.io`
+* `cdn03.quay.io`
+
+This change should be made to any firewall configuration that specifically enables outbound connections to `registry.redhat.io` or `registry.access.redhat.com`.
```

Comment:
Sounds like this is a required change. Some style guides (IBM, which we use along with RH Supplementary Guide) suggests using more direct words than "should".  See info for "should" in the [Word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s) section in the IBM Style guide.) Suggesting that this be changed to either "This change must be made to any firewall configuration..." or if you want to avoid passive voice in this case, "You must make this change to any firewall configuration that specifically enables outbound..."